tabipy.py

```diff
@@ -125,9 +141,9 @@ def __init__(self, value, **kwargs):
        super(TableHeader, self).__init__(value, header=True, **kwargs)
 
 class TableRow(object):
-    def  __init__(self, *cells, **kwargs):
+    def  __init__(self, *cells, max_len=None):
```

Comment:
Unfortunately this is not valid syntax in Python 2, and I don't think we want to drop support just yet. We'll need to go back to the kwargs.